src/Console/ModelsCommand.php

```diff
@@ -1096,6 +1096,9 @@ protected function hasCamelCaseModelProperties()
 
     protected function getAttributeReturnType(Model $model, \ReflectionMethod $reflectionMethod): Collection
     {
+        // Private/protected ReflectionMethods require setAccessible prior to PHP 8.1
+        $reflectionMethod->setAccessible(true);
```

Comment:
I chose not to wrap this in an if-statement checking for `PHP_VERSION_ID` since according to the docs from PHP 8.1 onward this method call is a noop.